docs/features/annotations.md

```diff
@@ -2,28 +2,50 @@
 
 Annotations is a feature introduced as part of the Speedgrader update to Autolab. It allows instructors and TAs to quickly leave comments and grade code at the same time. 
 
+!!! attention "Non-Autograded Problems Only"
+    Note that annotations can only be added to non-autograded problems. Specifically, a problem is non-autograded if there is no assigned score for that problem in the json outputted by the autograder
+
+### Line-based annotations
+
 ![Annotation Form](/images/annotations.png)
 
 Hover over any line of the code and click on the green arrow, and the annotation form will appear. Add the comment, adjust the score, and select the targeted problem.
 
-!!! attention "Non-Autograded Problems Only"
-    Note that annotations can only be added to non-autograded problems. Specifically, a problem is non-autograded if there is no assigned score for that problem in the json outputted by the autograder
+When the "Add to shared comment pool" option is checked, the comment will be saved to a shared comment pool.
```

Comment:
might want to add info about who the shared comment pool is shared with and whether the pool is per-assignment vs per-problem.

Reply:
Done, updated docs to clarify that shared comment pool is shared with other instructors and course-assistants, and that the pool is per assessment.